lib/LanguageServerRename/Model/Renamer/ChainRenamer.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Phpactor\Extension\LanguageServerRename\Bridge;
+namespace Phpactor\Extension\LanguageServerRename\Model\Renamer;
```

Comment:
As the ChainRenamer does not have any dependencies, it is not a bridge to anything -- it is part of the Model.